src/Files.Sdk.Storage/Extensions/StorageExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Files.Sdk.Storage.Extensions
+{
+    public static class StorageExtensions
+    {
+        public static async Task CopyContentsToAsync(this IFile source, IFile destination, CancellationToken cancellationToken = default)
```

Comment:
Should we add documentation/comments here?

Reply:
Yes, but we'll add those later on (perhaps). Those methods can be volatile and be removed/modified often